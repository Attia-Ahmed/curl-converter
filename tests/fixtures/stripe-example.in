curl -X POST https://api.stripe.com/v1/charges -u sk_test_sjp0J5IpIZ4o7L0OtmCW3s7p: -d amount=2000 -d currency=usd -d source=tok_visa -d "metadata[order_id]"=6735
