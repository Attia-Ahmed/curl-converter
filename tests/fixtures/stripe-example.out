Http::asForm()
    ->withBasicAuth('sk_test_sjp0J5IpIZ4o7L0OtmCW3s7p', '')
    ->post('https://api.stripe.com/v1/charges', [
        'amount' => 2000,
        'currency' => 'usd',
        'source' => 'tok_visa',
        'metadata' => [
            'order_id' => 6735,
        ],
    ]);
